### PR TITLE
Create environments.toml

### PR DESCRIPTION
**Pull request by D3vil0p3r**
Currently I don't think it is needed to add a description since all the needed info are in the README.md.

@phith0n the structure of each item appears like the following:
```toml
[[container]]
app = "ActiveMQ"
path = "activemq/CVE-2015-5254"

  [[container.name]]
  en = "Apache ActiveMQ Deserialization"
  zh = "Apache ActiveMQ 反序列化漏洞"

  [[container.cve]]
  id = "CVE-2015-5254"
  nvd = "https://nvd.nist.gov/vuln/detail/CVE-2015-5254"
```
In case one lab has more than one CVE, it should appear as:
```toml
[[container]]
app = "Struts2"
path = "struts2/s2-015"

  [[container.name]]
  en = "S2-015 Remote Code Execution"
  zh = "S2-015 远程代码执行漏洞"

  [[container.cve]]
  id = "CVE-2013-2134"
  nvd = "https://nvd.nist.gov/vuln/detail/CVE-2013-2134"
  
  [[container.cve]]
  id = "CVE-2013-2135"
  nvd = "https://nvd.nist.gov/vuln/detail/CVE-2013-2135"
```
So we have a table array named `container` and subtables for each property, for example `container.name` subtable will contain `en` and `zh` languages. Then. `container.cve` contains some info about CVE (currently id and NVD link). For those containers with no CVE, I just set as empty string `""` in order to not break any parsing by a custom script.

The properties outside the subtables should be at the beginning of the main table otherwise they will be seen as part of subtables.

Example of Python code:
```python
import toml

data = toml.load("/home/user/environments.toml")

for cve_lab in data['container']:
    print(cve_lab['name'][0]['en'])
    print(cve_lab['cve'])
    print(cve_lab['cve'][0]['id'])
    print(cve_lab['path'])
```

### CODE REVIEW

**Comment by phith0n** [comment]
Hi @D3vil0p3r 

I think the design is too difficult to understand.

My philosophy is to design a configuration that users can understand what each field does at first time without having to read any documentation. For example the current configuration is very simple:

```json
[{
        "name_en": "A simple CVE title",
        "name_zh": "A simple CVE title",
        "app": "example",
        "cve": "CVE-2023-0001",
        "path": "path/to/vuln"
    },...]
```

Since there is one lab has more than one CVE, can just use:

```
[{
        "name_en": "A simple CVE title",
        "name_zh": "A simple CVE title",
        "app": "example",
        "cve": ["CVE-2023-0001", "CVE-2023-0002"],
        "path": "path/to/vuln"
    },...]
```

NVD link is no need to exist in the configuration because you can simply build a NVD link from the CVE id like `'https://nvd.nist.gov/vuln/detail/' + env[0]['cve']`. Which is "Entities should not be multiplied unnecessarily".

In TOML format:

```
[[environments]]
name_en = "ActiveMQ Deserialization Vulnerabilities"
name_zh = "ActiveMQ 反序列化漏洞"
app = "ActiveMQ"
cve = ["CVE-2023-0001", "CVE-2023-0002"]
path = "path/to/vuln"
```

**Comment by phith0n** [comment]
Even I think we don't need to put Chinese title in the file, it's a previous design that I will slowly change.

**Comment by D3vil0p3r** [comment]
So it should appear like:
```toml
[[environments]]
name = "ActiveMQ Deserialization Vulnerabilities"
app = "ActiveMQ"
cve = ["CVE-2023-0001"]
path = "path/to/vuln"
```
I will define `cve` as a list of strings, also for cases with single CVE, in order to make it easier to script and get data.

**Comment by D3vil0p3r** [comment]
@phith0n fixed. Let me know if it is good.

**Comment by phith0n** [comment]
> I will define cve as a list of strings, also for cases with single CVE, in order to make it easier to script and get data.

Yep, we can leave it 3 types of CVE field: **string**, **array of strings** and **null**.

In my point of view, most of environments have only one CVE id so can just use **string** as default:

```toml
[[environment]]
name = "Apache ActiveMQ Deserialization"
cve = "CVE-2015-5254"
app = "ActiveMQ"
path = "activemq/CVE-2015-5254"
```

File: environments.toml [hunk 1]
```diff
@@ -0,0 +1,1409 @@
...
+
+[[environment]]
+name = "ThinkPHP5 5.0.22/5.1.29 Remote Code Execution"
+cve = [""]
+app = "ThinkPHP"
+path = "thinkphp/5-rce"
+
+[[environment]]
+name = "ThinkPHP5 5.0.23 Remote Code Execution"
+cve = [""]
```
Comment (phith0n) [inline environments.toml:1251]: just use `""` or `[]` to represent zero value, not `[""]`.

Comment (D3vil0p3r) [reply environments.toml]: @phith0n fixed.

**Comment by phith0n** [comment]
LGTM, thanks for your contribution.

**Comment by D3vil0p3r** [comment]
> LGTM, thanks for your contribution.

You are welcome. And thank you for your project. I integrated it on [Athena Cyber Hub](https://github.com/Athena-OS/athena-cyber-hub) Athena OS tool.

**Pull request merged**